ruff.toml

```diff
@@ -0,0 +1,76 @@
+# Enable rules.
+select = [
+    "A",
+    "C4",
+    "C90",
+    "COM",
+    #"D", # pydocstyle - not enabled temporarily
+    "E",
+    "ERA",
+    "FA",
+    "FIX",
+    "F",
+    "I",
+    "ICN",
+    "ISC",
+    "PLE",
+    "PLC",
+    "PLR",
+    "PLW",
+    "PL",
+    "PYI",
+    "Q",
+    "PTH",
+    "TD",
+    "W"
+]
+
+# Ignore specific rules
+ignore = [
+    "W291", # https://beta.ruff.rs/docs/rules/trailing-whitespace/
+    "PLR0913", # https://beta.ruff.rs/docs/rules/too-many-arguments/
+    "PLR2004", #https://beta.ruff.rs/docs/rules/magic-value-comparison/
+    "PLW0603", #https://beta.ruff.rs/docs/rules/global-statement/
+    "COM812", # Trailing comma missing - disabled temporarily
```

Comment:
When we enable this check, we will potentially have 494 errors to fix. I suggest creating a specific PR for this.